recipe/meta.yaml

```diff
@@ -8,14 +9,21 @@ package:
 
 source:
   fn: SuiteSparse-{{ version }}.tar.gz
-  url: http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-{{ version }}.tar.gz
-  sha256: 698b5c455645bb1ad29a185f0d52025f3bd7cb7261e182c8878b0eb60567a714
+  url: http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-{{ version }}.tar.gz  # [not win]
+  sha256: 698b5c455645bb1ad29a185f0d52025f3bd7cb7261e182c8878b0eb60567a714  # [not win]
+  url: https://github.com/jlblancoc/suitesparse-metis-for-windows/archive/{{ version_win }}.tar.gz  # [win]
+  sha256: 719065607201f445d490b6bb7bb43df2e8f1affb187551c841d5d2a3dd72f2e7  # [win]
+  patches:
+    - gh_53.patch  # [win]
+    - gh_54.patch  # [win]
 
 build:
-  skip: true  # [win]
+  skip: true  # [win32]
   number: 201
   features:
     - blas_{{ variant }}
+    - vc14    # [win]
+  msvc_compiler: 14.0
```

Comment:
We shouldn't need both of these. `vc14` should be fine.

Reply:
This was incorrect advice.

Reply:
Normally we just use `python` in this case and it works fine with features. That was what I was suggesting.